### PR TITLE
feat(buck): adds mv path command

### PR DESCRIPTION
**Pull request by andrewxhill**
waiting for protos

### CODE REVIEW

File: packages/buck-util/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@textile/buck-util",
-  "version": "1.0.4",
-  "description": "NodeJS utiliting for pushing and pulling buckets.",
+  "version": "1.0.5-alpha.0",
+  "description": "This library has been replaced by @textile/buckr. This library is deprecated.",
```
Comment (andrewxhill) [inline packages/buck-util/package.json:4]: name always bugged me so took the chance to get rid of this old one

File: packages/buckets/package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "@improbable-eng/grpc-web": "^0.13.0",
     "@repeaterjs/repeater": "^3.0.4",
-    "@textile/buckets-grpc": "2.6.0",
+    "@textile/buckets-grpc": "2.6.4",
```
Comment (andrewxhill) [inline packages/buckets/package.json:25]: latest

File: packages/buckets/src/api/index.ts [hunk 3]
```diff
@@ -401,6 +402,24 @@ export async function bucketsListIpfsPath(
   return fromPbPathItemNullable(res.getItem())
 }
 
+/**
+ * Move a file or subpath to a new path.
+ * @internal
+ */
+export async function bucketsMovePath(
```
Comment (andrewxhill) [inline packages/buckets/src/api/index.ts:409]: does what it says

File: packages/hub/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@textile/hub",
-  "version": "6.0.2",
+  "version": "6.0.3-alpha.0",
```
Comment (andrewxhill) [inline packages/hub/package.json:3]: all these just came from pushing an rc build changes to this. should autofix on next publish

File: packages/buckr/package.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@textile/buckr",
+  "version": "1.0.0",
+  "description": "NodeJS utility for working with remote buckets.",
```
Comment (andrewxhill) [inline packages/buckr/package.json:4]: pretty nice now with a few more commands. will use in npm scripts for a few more things (e.g. filecoin archives)

File: packages/buckr/src/index.ts [hunk 6]
```diff
@@ -0,0 +1,339 @@
+import fs from 'fs'
```
Comment (andrewxhill) [inline packages/buckr/src/index.ts:1]: did this as an experiment the other day, but figured we might as well capture these methods. if nothing else they serve as good usage examples for typescript.

File: packages/buck-util/package.json (hunk 1 above)
Comment (carsonfarmer) [reply packages/buck-util/package.json:4]: LOL, cool!

Comment (andrewxhill) [reply packages/buck-util/package.json:4]: buckr up!

**Review by carsonfarmer — APPROVED** [review]
LGTM!

**Pull request merged**